plugins/inlinetoolbar/plugin.js

```diff
@@ -151,36 +184,43 @@
 			};
 
 			/**
-			 * Places the inline toolbar next to a specified element so the tip of the toolbar's triangle
-			 * points to that element.
-			 *
+			 * @inheritdoc CKEDITOR.ui.balloonPanel#attach
 			 * @member CKEDITOR.ui.inlineToolbarView
-			 * @param {CKEDITOR.dom.element} element The element to which the panel is attached.
 			 */
-			CKEDITOR.ui.inlineToolbarView.prototype.create = function( element ) {
-				this.attach( element, false );
-
-				var that = this,
-					editable = this.editor.editable();
-				this._detachListeners();
-
-				this._listeners.push( this.editor.on( 'resize', function() {
-					that.attach( element, false );
-				} ) );
-				this._listeners.push( editable.attachListener( editable.getDocument(), 'scroll', function() {
-					that.attach( element, false );
-				} ) );
+			CKEDITOR.ui.inlineToolbarView.prototype.attach = function( element, options ) {
+				this._element = element;
```

Comment:
This member was not documented.

Additionally please, find more specific name for it. How about `_pointedElement`? Var name needs to have a meaning, `element` might be some layout part as well.